readme.md

```diff
@@ -71,7 +71,7 @@ here's a workaround, add this to your pom.xml:
 - get as java.util.Properties convenience
 
 ## Change History
-
+- 0.4.2 toProperties convenience method, transformer interface for java to transform from string to any time
```

Comment:
To any time? Or any type?

Reply:
type. protip: don't write these things too tired :p